### PR TITLE
Remove map component and fix styling

### PR DESCRIPTION
**Pull request by cristian-aldea**
I have removed the map component and moved the MapView to the map screen to simplify the styling and logic.

I fixed the campus toggle styling, however, the styling for the sliding panel might still be broken since it is currently disabled.

Issue:  #32

### CODE REVIEW

**Review by RGPosadas — DISMISSED** [review]
LGTM!

File: components/campus-toggle/campus-toggle.component.jsx [hunk 1]
```diff
@@ -75,41 +75,42 @@ const CampusToggle = () => {
 /**
  * TODO: styling might need to be changed @Ragith
```
Comment (osahtout) [inline components/campus-toggle/campus-toggle.component.jsx]: Please remove comments

**Review by osahtout — CHANGES_REQUESTED** [review]
Please add comments to explain what you did.

**Review by bit172 — DISMISSED** [review]
88261643_228863368280138_524319265214431232_n
LGTM!

**Review by osahtout — DISMISSED** [review]

**Review by RGPosadas — APPROVED** [review]
Nice addition of the comments, extra LGTM

**Pull request merged**